ui-lib/src/main/res/ui/restore/values/strings.xml

```diff
@@ -211,12 +235,15 @@
     <string name="ns_format_date_time">dd/MM/yyyy</string>
     <string name="ns_view_all_transactions">View transaction history</string>
     <string name="ns_downloading_block">Downloading\n\nBlock %1s of %2s</string>
+    <string name="ns_downloading_wallet">Downloading…%1d%%</string>
```

Comment:
Remove “Downloading” strings as it is no longer a separate activity.

Reply:
Sure, I will do the refactor of strings file again feature wise there I will remove these extra strings